themes/OpenFisca-2022/layouts/partials/card.html

```diff
@@ -5,12 +5,15 @@
 	{{ $imageSrc = $newImageSrc}}
 {{ end }}
 
-<div class="card {{ if .website }}card-hasLink{{ end }}">
-	<div class="card_image">
-		<img class="screenshot" src="{{ $imageSrc }}" alt="{{ .title }}" />
-	</div>
+<div class="card{{ if .website }} card-hasLink{{ end }}{{ with .until }} card-isUntil{{ end }}">
+	{{ if not .until }}
```

Comment:
Why not show the image even for retired products? 🙂

Reply:
It didn't make much sense for me to put a small preview image of a reuse that is no longer accessible.